conda-forge.yml

```diff
@@ -4,6 +4,7 @@ azure:
       CONDA_BLD_PATH: C:\bld\
 build_platform:
   osx_arm64: osx_64
+  linux_aarch64: linux_64
```

Comment:
Is cross compiling now supported! So much easier to build!

Reply:
On that note, shall we try cross-compiling for ppc?

Reply:
probably after this PR is in.